index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=1024px, initial-scale=1.0">
+  <title>Valentina Krivelskaya</title>
+</head>
+<body>
+  <header>
+    <h1>Valentina Krivelskaya</h1>
+    <address>
+      <div>Email: v.krivelskaya@mail.ru</div>
+      <div>Address: 54-234 Belinskogo str, Minsk, Belarus</div> 
+      <div>Phone: (29) 543 38 93</div>      
+       </address>    
+  </header>
+  
+</body>
+<img src="image.jpg" width="100px" alt="There are only tags in my mind">
```

Comment:
I'm not able to check it right now, but I'm pretty sure that it's your profile photo. 
In that case alt should be for example 'Profile photo' :)